lib/matplotlib/__init__.py

```diff
@@ -173,59 +174,39 @@ def compare_versions(a, b):
     else:
         return False
 
+
+try:
+    import dateutil
+except ImportError:
+    raise ImportError("Matplotlib requires dateutil")
+
+
 if not compare_versions(six.__version__, '1.3'):
     raise ImportError(
-        'six 1.3 or later is required; you have %s' % (
-            six.__version__))
+        "Matplotlib requires six>=1.3; you have %s" % six.__version__)
```

Comment:
This isn't even true; I always have trouble running Matplotlib through Pycharm's profiler because it has six 1.9.